integreatly-docs/modules/ROOT/pages/index.adoc

```diff
@@ -63,14 +62,17 @@ include::_partials/ref_terminology.adoc[leveloffset=+1,tags={include-tags}]
 include::_partials/obtaining_firebase_cloud_messaging_credentials.adoc[leveloffset=+2,tags={include-tags}]
 include::_partials/enable_push_notifications_for_ios_app.adoc[leveloffset=+2,tags={include-tags}]
 [[setting-up-the-push-client-app]]
-== Configuring your client app
-//include::_partials/downloading-configuration-file.adoc[leveloffset=+2,tags={include-tags}]
-include::_partials/setting-up-the-push-sdk.adoc[leveloffset=+2,tags={include-tags}]
-include::_partials/registering_device.adoc[leveloffset=+1,tags={include-tags}]
-include::_partials/sending-a-push-notification.adoc[leveloffset=+1,tags={include-tags}]
-include::_partials/handling-push-notifications.adoc[leveloffset=+1,tags={include-tags}]
 
+== Configuring the Push Notification Service
+
+Access the Push Notification Service section by clicking *Push Notification Service* under the *All Services* tab from the {webapp}.
 
-// Restore the context to what it was before this assembly.
-ifdef::parent-context[:context: {parent-context}]
-ifndef::parent-context[:!context:]
+* If some applications already exist, they will be displayed so click "Create Application" to go to the Welcome page.  If there are no previous applications, you will be taken directly to the Welcome page.
```

Comment:
should be numbered list:
![image](https://user-images.githubusercontent.com/5154224/71090469-f9ed9a80-219a-11ea-8743-a53cd5abec06.png)


period instead of asterix

Reply:
Changed.